app/transport/http/const.go

```diff
@@ -1,8 +1,9 @@
 package http
 
 const (
-	MsgUserCreated      = "User created successfully"
-	MsgEmailConflict    = "Email is already taken"
-	MsgInternalSeverErr = "Internal server error"
-	MsgBadRequest       = "Bad request"
+	MsgUserCreated   = "User created successfully with the following ID"
+	MsgEmailConflict = "Email is already taken"
+	MsgValidationErr = "Validation error"
+	MsgBadRequest    = "Provided JSON file is invalid"
```

Comment:
First of all, the request is not a file. File - it's a different Content-Type, and we don't use it, we use text body in the form of JSON.
Also not each BadRequest situation is an invalid JSON. For example, invalid email - is a BadRequest situation, however it's not an invalid JSON situation.

Reply:
Absolutely right, thank you! Fixed